_pages/tools.md

```diff
@@ -49,6 +50,5 @@ Page under construction.
 
 - [ArchSAT](https://github.com/Gbury/archsat): SMT solver generating Dedukti/FOL
 - [cvc5](https://cvc5.github.io/): efficient versatile SMT solver generating proof traces
-- [z3](https://github.com/Z3Prover/z3/wiki): efficient SMT solver that generates [proofs](https://stackoverflow.com/questions/29577754/getting-proof-from-z3py) (for a subset of checks, at least).
```

Comment:
Will squash-merging this commit remove z3 that you've just added in a separate pull request?

Reply:
My bad, I was doing this from the web interface. I believe merging this without being in interactive mode will remove z3. 

We can delete this PR and I'll do a new one without z3.